src/utils/build.utils.js

```diff
@@ -94,7 +94,7 @@ const BuildUtils = {
     return uniqueMissingModules
   },
 
-  async buildPackage({ name, installPath, externals, options }) {
+  async buildPackage({ name, installPath, externals, isLocal, options }) {
```

Comment:
Yeah, I agree that passing `isLocal` around isn't the best solution, but I was concerned that I miss an edge case if I tried to test the path and break the normal functionality. I don't have the name of the package in this method (I should double check `stats` though), so the only other way I think of is to test whether `node_modules` under the install path has a single symbolic link as its only child. In that case, it would have to be a local install. Any objections to this? I can get a solution out tomorrow.

Reply:
Let's pass the package name from `build.utils.js#buildPackage` to `getDependencySizeTree.js#bundleSizeTree` and remove prefixes until first occurrence.